site/en/docs/privacy-sandbox/attribution-reporting-event-introduction/index.md

```diff
@@ -0,0 +1,321 @@
+
+#### Example
+
+If a user clicks an ad and converts three days later, you may receive the report about seven days
+later.
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+Report scheduling: detail
+{% endDetailsSummary %}
+
+In Chrome, report scheduling for clicks works as follows:
+
+<div class="w-table-wrapper"><table class="w-table--top-align"><thead><tr><th>Value of <code >attributionexpiry</code><br></th><th>Depending on the conversion time, a report is sent (if the browser is open)...</th><th>Number of reporting windows</th></tr></thead><tbody><tr><td>30 days (the default and maximum value)</td><td><ul><li>2 days after the ad was clicked</li><li>or 7 days after click</li><li>or ((<code>attributionexpiry</code>= 30 days) + 1 hour) after click.</li></ul></td><td>3</td></tr><tr><td>Between 7 and 30 days</td><td><ul><li>2 days after click</li><li>or 7 days after click</li><li>or (<code>attributionexpiry</code> + 1 hour) after click.</li></ul></td><td>3</td></tr><tr><td>Between 2 and 7 days</td><td><ul><li>2 days after click</li><li>or (<code>attributionexpiry</code> + 1 hour) after click.</li></ul></td><td>2</td></tr><tr><td>Under 2 days</td><td><ul><li>(2 days + 1 hour) after click</li></ul></td><td>1</td></tr></tbody></table></div>
+{% endDetails %}
+ 
+### Attribution model
+ 
+By default, the attribution model is **last-touch**: for a given conversion, only the last
+engagement with an ad will generate a report.
+ 
+You can change this behavior by configuring [source-side priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#define-custom-attribution-rules).
+ 
+#### Example
+ 
+A user visits `news.example` and clicks an ad for `shoes.example`. On the next day, they visit `other-news.example` and again click an ad for `shoes.example`. A few days later, they visit `shoes.example` and convert by making a purchase.
+
+By default, unless you configure custom [source-side priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#define-custom-attribution-rules), you'll receive a report that attributes this conversion to the ad clicked on `other-news.example`.
+
+### Matching source and trigger data
+
+Let's take a given ad click (source event) with an click ID (source event ID) of `1298765678762`.
+When this click is registered, the browser⏤on the user's device⏤stores the click ID `1298765678762`, and alongside it:
+
+- `attributiondestination`. Typically, this is an advertiser site. Example: `shoes.example`.
+- `attributionreportto`. Typically, this is an adtech endpoint. Example: `adtech.example`.
+
+Later on, a trigger (conversion) takes place on a page.
+The browser will match this trigger with the source event ID `1298765678762` in the following conditions:
+
+- The page on which the trigger (conversion) took place matches **the eTLD+1 that was provided in the source's `attributiondestination`**. Example: `shoes.example/checkout`.
```

Comment:
Good place to link to https://web.dev/same-site-same-origin/#site ?

Reply:
Indeed! done